crates/pathfinder/src/consensus/inner/p2p_task.rs

```diff
@@ -73,10 +80,14 @@ pub fn spawn(
                         Event::Proposal(height_and_round, proposal_part) => {
                             if let Ok(Some((proposal_commitment, proposer))) =
                                 handle_incoming_proposal_part(
+                                    chain_id,
```

Comment:
`handle_incoming_proposal_part` fails at height `10` if we run a manual test with an empty DB:
```
  2025-08-29T16:51:40  WARN pathfinder_lib::consensus::inner::p2p_task: Error handling incoming proposal part for (10:0): Error {
    context: "Creating BlockExecutor",
    source: Getting historical block hash,
}
```
We need to add a match statement here and at least log the error as now it is just ignored.